### ISSUE TITLE
Dome light - color incorrect on lightmaps?

### ISSUE THREAD

**Issue by legoboyvdlp** [c172p-team/c172p#1158]
Hi there,
I noticed that with the dome light (only it) on, there is red light on the pedestal:

![image](https://user-images.githubusercontent.com/10434911/43518656-a41ec9f6-9584-11e8-855c-8089cbfb8ab3.png)

as opposed to 
![image](https://user-images.githubusercontent.com/10434911/43518668-ad0ebefe-9584-11e8-8826-611af21184d0.png)

Maybe I'm missing something, but shouldn't the light be white?

**Comment by legoboyvdlp** [comment]
I'm not sure, but I think the same is seen on the yoke with the dome light.

**Comment by gilbertohasnofb** [comment]
I will test this when I get home later tonight and then report back if no one beats me to it.

**Comment by wlbragg** [comment]
I tested this and here is what I noticed. At night, it is correct, red with the flood and yellow with the dome. In the daylight it does appear to take on more of a red tint. I think it is because of the added light to the scene. I don't think it is anything that needs address for a couple reasons.
One it is the rendering systems interpretation of the scene lighting conditions.
It is barely noticeable and only if you have the dome light on in the daytime (not a common thing to do).
It may actually be the correct effect due to the conditions.
